Feature/jse 45 pallete setup

AubreyMartin opened the pull request:
Feature/JSE-45 Palette Setup

Implemented palette setup in theme.ts with new and updated color definitions. 

Configured primary, secondary, backgroundAlt, secondaryLight, gray, warning, and accent colors.

AubreyMartin committed:
Update Palette Colors

Corrected color definitions: updated primary and secondary colors,
and added new backgroundAlt, secondaryLight, gray, warning, and accent colors.

AubreyMartin committed:
Added and updated color definitions in theme.ts: configured primary, secondary, backgroundAlt, secondaryLight, gray, warning, and accent colors.

AubreyMartin committed:
Fix: Naming Convention and Configuration Issue

AubreyMartin committed:
Fix: Extend Palette and TypographyVariants to include custom colors and body3 typography; add necessary imports for theme configuration.

AubreyMartin committed:
Your commit message

AubreyMartin committed:
Added Color Tiles Component

 1.Implemented ColorTileGrid component to display color tiles in a grid layout.
 2.Each tile shows the color name and its corresponding hex code, arranged vertically

AubreyMartin committed:
Remove unused React import from App.tsx to resolve TypeScript error TS6133.

AubreyMartin committed:
Updated font colors across components

AubreyMartin committed:
alined the font

judemartin commented on src/App.tsx:
```diff
@@ -1,117 +1,262 @@
+              alignItems: "center",
+              justifyContent: "bottom",
+              backgroundColor: "#62BCFA", color: "#FFFFFF",
+            }}
```
Hey @AubreyMartin , can we pull these colors from the theme.ts

judemartin commented on src/App.tsx:
```diff
@@ -1,117 +1,262 @@
-        H4 - Jost Bold | 14px; 20px Line; -0.2 Spacing <br /> Ut scelerisque
-        hendrerit tellus. Integer sagittis{" "}
-      </Typography>
+        <Grid container spacing={2}>
```
instead of div, please use Box

AubreyMartin replied:
Sure! I'll update the code to use Box instead of div. Thanks for the suggestion!

judemartin commented on src/App.tsx:
```diff
@@ -1,117 +1,262 @@
+        <Grid item xs={3}>
+          <Paper
+            style={{
+              height: 100,
```
please let's use sx props https://mui.com/system/getting-started/the-sx-prop/

AubreyMartin committed:
Here's a GitHub commit message you can use:

feat: Replace div with Box, apply sx props, and add white color to the theme palette

Replaced div elements with Box from Material-UI for better styling flexibility.
Utilized sx props for consistent and theme-based styling.
Added white color to the palette in theme.ts for centralized management.

AubreyMartin committed:
Replace div with Box, apply sx props, and add white color to the theme palette

Replaced div elements with Box from Material-UI for better styling flexibility.
Utilized sx props for consistent and theme-based styling.
Added white color to the palette in theme.ts for centralized management.

AubreyMartin committed:
Fix: Corrected backgroundColor property in Paper component using theme.palette.info.main for compatibility.

AubreyMartin committed:
fixed info Error

AubreyMartin commented:
@judemartin updated the changes 
![image](https://github.com/user-attachments/assets/f97e90f9-f47b-4e6c-8d50-c264be76e02b)

judemartin reviewed (approved):
Great work 💯

judemartin commented on src/App.tsx:
```diff
@@ -1,117 +1,259 @@
+import { Button, Stack, Typography, Grid, Paper } from "@mui/material";
 import "./App.css";
+import { Box } from "@mui/system";
 
```
nit : import Box from the same `import { Button, Stack, Typography, Grid, Paper } from "@mui/material"; and not from system

AubreyMartin committed:
Updated import Box from the same `import { Button, Stack, Typography, Grid, Paper } from "@mui/material";
—
